stellargraph/datasets/datasets.py

```diff
@@ -43,7 +43,29 @@ class Cora(
     "indicating the absence/presence of the corresponding word from the dictionary. The dictionary consists of 1433 unique words.",
     source="https://linqs.soe.ucsc.edu/data",
 ):
-    pass
+    def load(self):
```

Comment:
I wonder if it would be useful to offer an option here to only return the largest connected component and also whether the user wants a directed or undirected graph. Even though Cora is a directed graph, it is often used as an undirected one. 

In the current notebooks we load Cora as undirected. I run the GCN node classification notebook a few times and it looks like it performs better, e.g. higher accuracy on the test set, when the graph is undirected. 

Unless it is easy for users to cast a StellarDiGraph object to StellarGraph, we should give them the option in the load method.

Reply:
(Worked started on #948 at https://github.com/stellargraph/stellargraph/pull/958.)